src/@ionic-native/plugins/gao-de-location/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+
+/**
+ * @name Gao De Location
+ * @description
+ * Get the current location
```

Comment:
Please provide a better description. I'm not that familiar with Chinese so I don't know the difference between this plugin and the Cordova geolocation plugin.